dev-packages/eslint-plugin/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "private": true,
+  "name": "@theia/eslint-plugin",
+  "version": "1.11.0",
+  "description": "Custom ESLint rules for developing Theia extensions and applications",
+  "peerDependencies": {
+    "@theia/core": "1.11.0"
```

Comment:
I tried bumping versions, and lerna apparently doesn't touch peer dependencies :(

I changed this to be a regular dependency instead. It will work the same way it does for Theia extensions then.